op_map.cc

```diff
@@ -391,6 +391,46 @@ struct OpMapperBase : public vx::op_map::IOpMapper {
       const void* params) {
     return false;
   }
+
+  std::vector<int32_t> Extend_Broadcast(const std::shared_ptr<tim::vx::Tensor>& base_shape_tensor,
```

Comment:
name conversion defeat:: function name should not with "_"

Reply:
Renamed